### PR TITLE
KCD: Create INI files and mods folder as needed

### PR DESCRIPTION
**Pull request by LostDragonist**

### CODE REVIEW

**Comment by Holt59** [comment]
I'm not really pro-this, the issue is about mapping files that do not exists, right? As long as it's on the MO2 side, we can handle it, but if we present games with empty files instead of non-existing this could break havoc that's hard to investigate for MO2 users.

**Comment by LostDragonist** [comment]
The issue is that an exception is thrown and MO refuses to load

File: games/game_kingdomcomedeliverance.py [hunk 1]
```diff
@@ -22,3 +25,18 @@ class KingdomComeDeliveranceGame(BasicGame):
...
+            if not os.path.exists(iniPath):
+                with open(iniPath, "w") as _:
+                    pass
+
+        # Create the mods directory if it doesn't exist
+        modsPath = self.dataDirectory().absolutePath()
+        if not os.path.exists(modsPath):
+            os.mkdir(modsPath)
+
+        super(KingdomComeDeliveranceGame, self).initializeProfile(path, settings)
```
Comment (Holt59) [inline games/game_kingdomcomedeliverance.py:42]: `super()` is sufficient here.

Comment (LostDragonist) [reply games/game_kingdomcomedeliverance.py]: done

**Pull request merged**